### DISCUSSION TITLE
Issue - LNK4217

### DISCUSSION THREAD

**Discussion by bxqgit** [krzyzanowskim/CryptoSwift#1003] [Q&A]
I'm getting the following warning while building a project:

```lld-link: warning: C:\Foo\.build\x86_64-unknown-windows-msvc\debug\Foo.build\Crypto.swift.o: locally defined symbol imported: $s11CryptoSwift6DigestV3md5ySays5UInt8VGAGFZ (defined in C:\Foo\.build\x86_64-unknown-windows-msvc\debug\CryptoSwift.build\Digest.swift.o) [LNK4217]```

Any ideas how to fix it? 🤔

**Comment by krzyzanowskim** [comment]
It's a linker warning. It looks to me like a Windows tooling related thing, not really something I can address.

**Reply by bxqgit** [reply]
The warning disappears if `CryptoSwift` is removed from the dependencies.

Anyway, is there a way to turn off this warning?
